### ISSUE TITLE
`k initialized to invalid value (-nan)` in logistic growth. 

### ISSUE THREAD

**Issue by oshribr** [facebookincubator/prophet#258]
Hi all, 
When I'm using  prophet with `growth == 'logistic'` I get the following error
`k initialized to invalid value (-nan)`
After a little debug I found the issue. 
The issue raises in case  the `y` of the minimum date  is half of the `y` max, and the `df['cap_scaled'] == 1`. 
the following line assigns `2` into `r0`
```
r0 = max(1.01, df['cap_scaled'].ix[i0] / df['y_scaled'].ix[i0])
```
and this line assigns `0` into `L0`
L0 = np.log(r0 - 1)

this line assigns `0` into m 

`m = L0 * T / (L0 - L1)`

and this line divides by zero 

`k = L0 / m`

How  should I fix this? 
Is the following workaround good:
The workaround I thought about is to subtract or add epsilon to the `y` in the min date.

**Comment by oshribr** [comment]
I changed the `k = L0 / m` to  `k = (L0 - L1) / T`

**Comment by bletham** [comment]
Thanks, yeah I came across this for the first time just a couple weeks ago and pushed the fix to the v0.2 branch in https://github.com/facebookincubator/prophet/commit/ff1dbcf38d2258420630bfb1dc6998cfdd24a9b2. A math bug :-)

**Comment by bletham** [comment]
This fix has been pushed in v0.2 on CRAN and pypi.
